### ISSUE TITLE
Fails to browse folders if the path name is too long

### ISSUE THREAD

**Issue by FM1337** [Cpasjuste/pplay#1]
I tested the network part of the player, and if a path is too long, it will refuse to browse to that folder.

I tested a path with about 85 characters in total.

**Comment by Cpasjuste** [comment]
Thanks for the report, i'll look into it :)

**Comment by FM1337** [comment]
It also won't list files that cause the path to become too long

**Comment by Cpasjuste** [comment]
Noted. I guess it should be easy to fix.

**Comment by Cpasjuste** [comment]
Hi @FM1337, i think the problem is not in the path length (at least for the browsing part, i fixed the media information caching of long path). It may be caused by special characters in it?

**Comment by FM1337** [comment]
Yeah that could also be the cause, any way it can be fixed on the app's
side?

**Comment by FM1337** [comment]
Any updates on this @Cpasjuste? I still seem to run into this issue

**Comment by Cpasjuste** [comment]
May be fixed with 2.1?

**Comment by FM1337** [comment]
Nope that just hides them now.

**Comment by Cpasjuste** [comment]
Can you provide me an example of a non working path?

**Comment by FM1337** [comment]
/media/TV/classic stuff/1982/action/Knight Rider/Season 1/Knight.Rider.S01E01 Knight Of the Phoenix Part 1.BluRay.1080p.mkv is an example.

**Comment by FM1337** [comment]
it should be noted that I only tried this with the http server, I haven't tried ftp

**Comment by FM1337** [comment]
This issue doesn't appear to affect FTP
